pluto_build/sql/qaqc_outlier.sql

```diff
@@ -0,0 +1,51 @@
+-- -v VERSION=$VERSION
+CREATE TABLE IF NOT EXISTS qaqc_outlier(
+    v character varying,
+    CONDO character varying,
+    MAPPED character varying,
+    outlier character varying
+);
+
+DELETE FROM qaqc_outlier
+WHERE v = :'VERSION'
+AND CONDO::boolean = :CONDO
+AND MAPPED::boolean = :MAPPED;
+
+INSERT INTO qaqc_outlier (
+select :'VERSION' as v, 	 
+       :CONDO as condo,
+       :MAPPED as mapped,
+       jsonb_agg(t) as outlier
+from (
+	select jsonb_agg(json_build_object('bbl', tmp.bbl, 'unitsres',tmp.unitsres, 'resarea', tmp.resarea,
+                              'res_unit_ratio', tmp.res_unit_ratio)) as values, 'unitsres_resarea' as field
+	from (SELECT bbl, unitsres, resarea, resarea::FLOAT/unitsres::FLOAT as res_unit_ratio
+          FROM (SELECT a.* 
+                FROM archive_pluto a
+                :CONDITION) b
+          WHERE unitsres::FLOAT>50 and resarea::FLOAT/unitsres::FLOAT < 300
+          AND resarea::FLOAT!= 0)tmp
+	union
+	select jsonb_agg(json_build_object('bbl', tmp.bbl, 'bldgarea',tmp.bldgarea, 'lotarea', tmp.lotarea,
+                              'numfloors', tmp.numfloors, 'blog_lot_ratio', tmp.bldg_lot_ratio)) as values, 'lotarea_numfloor' as field
+	from (SELECT bbl, bldgarea, lotarea, numfloors, bldgarea::FLOAT/lotarea::FLOAT as bldg_lot_ratio
+          FROM (SELECT a.*
+                FROM archive_pluto a
+                :CONDITION) b
+          WHERE bldgarea::FLOAT/lotarea::FLOAT > numfloors::FLOAT*2
+          AND lotarea::FLOAT != 0 AND numfloors !=0)tmp
+    union 
+    select jsonb_agg(json_build_object('pair',m.pair, 'bbl', m.bbl,
+                                       'building_area_current',m.building_area_current,
+                                       'building_area_previous',m.building_area_previous,
+                                       'percent_change',m.percent_change)) as values, 
+                                       'building_area_increase' as field
+    from (SELECT :'VERSION'||' - '||:'VERSION_PREV' as pair,
+                a.bbl, a.lotarea::FLOAT as building_area_current, b.lotarea::FLOAT as building_area_previous, 
+                (a.lotarea::FLOAT-b.lotarea::FLOAT)/b.lotarea::FLOAT as percent_change
```

Comment:
This is almost there, need to multiple by 100 to get into units of percentage or change field name.
I also wanted to bring up my own confusion with the term "percent change". I looked at the numbers and the first row in the table is bbl `1002410018`. The current area is `326512`, and the previous `137575`. The current is 2.373 times the previous. I see that you are using [this formula for percent change](https://en.wikipedia.org/wiki/Relative_change_and_difference#Percentage_change) which gets the answer of 137.3% percent change. 
This seems to be technically correct, but I want to flag that to me a 137.3% "percent change" means going from 100 to 137.3. Whereas using the formula on wikipedia, it means adding 137.3% of the original total to go to 237.3. I wanted to bring @NYCPlanning/data-engineering's attention this question so we can get on the same page about what a "percent change" means (we should probably try to stick to the correct definition like Jingyi does here).

Reply:
okay, this makes sense. As we are sorting out all BBLs that increased more than doubled than previous version, which means the percent change (in my definition or wiki) should be greater than 100%.